.env.local

```diff
@@ -0,0 +1,6 @@
+COSMIC_BUCKET_SLUG=vuetify-vslidegroup
+COSMIC_READ_KEY=5FIzwbW5bEHpNLRcL4U0yELZAvHv8HVCd0DCUeiNUtA7OJKTgg
+COSMIC_PREVIEW_SECRET=fK2g0BBjfP6gkfX4FyTcID2RsJzvLdBpDJCYRV8Rgg3cUtvjlq
```

Comment:
I'm guessing you don't want to commit those, or want this merged 😨

Reply:
I DID NOT want to. Thank you.